core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/hybrid/HybridJDBCConstants.java

```diff
@@ -161,4 +161,13 @@ public class HybridJDBCConstants {
 
     public static final String COUNT_INTERNAL_ONLY_ROLES_SQL = "SELECT COUNT(UM_ID) AS RESULT FROM UM_HYBRID_ROLE " +
             "WHERE UM_ROLE_NAME NOT LIKE 'Application%' AND UM_ROLE_NAME LIKE ? AND UM_TENANT_ID = ?";
+
+    public static final String GET_GROUP_ID = "SELECT UM_ID FROM UM_HYBRID_GROUP_ROLE WHERE UM_GROUP_NAME =? " +
```

Comment:
`GET_GROUP_ID` -> `GET_GROUP_ROLE_MAPPING_ID`
This query doesn't return the group id right? it's the mapping id

Reply:
Fixed with b7dc7f9a243894142c16de500aee222576707e5a